test/node/features/fixtures/contextualize/scenarios/contextualize.js

```diff
@@ -11,6 +11,7 @@ Bugsnag.start({
 var contextualize = Bugsnag.getPlugin('contextualize')
 
 contextualize(function () {
+  console.log('manual notify')
```

Comment:
I know it's only a test - but are these logs being used?

Reply:
yes, ensuring that a breadcrumb is created on console.log

Reply:
see `test/node/features/contextualize.feature`